panedr/requirements.txt

```diff
@@ -1,3 +1,4 @@
 pandas
 pyedr
 pbr
+tqdm
```

Comment:
Just FYI here - @BFedder and I had a chat about this a week or so ago and I told him to go ahead with this, tqdm is a reasonably small dependency, and it will make life a lot easier imho.

@jbarnoud as the original code author, do you have any issues with this?